### PR TITLE
Update context.

### PR DESCRIPTION
**Pull request by metanerd**
Summary

Update circleci context and add new s3 buckets.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -104,14 +104,14 @@ workflows:
...
-          s3-copy-to: s3://releases.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/
+          s3-copy-to: s3://pr-builds.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/
           context: mattermost-ci-s3
           requires:
             - build-ltops-and-test
           <<: *only_branches
       - build-docker:
           <<: *mm_binary
-          loadtest-binary: https://releases.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/mattermost-load-test.tar.gz
+          loadtest-binary: https://pr-builds.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/mattermost-load-test.tar.gz
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          loadtest-binary: https://pr-builds.mattermost.com/loadtest/${CIRCLE_BRANCH}/mattermost-load-test.tar.gz
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -104,14 +104,14 @@ workflows:
 
       - upload-s3:
           name: upload-s3-branches
-          s3-copy-to: s3://releases.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/
+          s3-copy-to: s3://pr-builds.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          s3-copy-to: s3://pr-builds.mattermost.com/loadtest/${CIRCLE_BRANCH}/
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -136,14 +136,14 @@ workflows:
 
       - upload-s3:
           name: upload-s3-tags
-          s3-copy-to: s3://releases.mattermost.com/mattermost-load-test/${CIRCLE_TAG}/
-          context: mattermost-ci-s3
+          s3-copy-to: s3://pr-builds.mattermost.com/mattermost-load-test/${CIRCLE_TAG}/
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          s3-copy-to: s3://releases.mattermost.com/mattermost-load-test/${CIRCLE_TAG}/
```

File: .circleci/config.yml [hunk 4]
```diff
@@ -136,14 +136,14 @@ workflows:
...
+          context: mattermost-ci-pr-builds-s3
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          context: mattermost-release-s3
```

File: .circleci/config.yml [hunk 5]
```diff
@@ -136,14 +136,14 @@ workflows:
...
           <<: *only_v_tags
           requires:
             - build-ltops-and-test
       - build-docker:
           <<: *mm_binary
-          loadtest-binary: https://releases.mattermost.com/mattermost-load-test/${CIRCLE_TAG}/mattermost-load-test.tar.gz
+          loadtest-binary: https://pr-builds.mattermost.com/mattermost-load-test/${CIRCLE_TAG}/mattermost-load-test.tar.gz
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          loadtest-binary: https://releases.mattermost.com/mattermost-load-test/${CIRCLE_TAG}/mattermost-load-test.tar.gz
```

File: .circleci/config.yml [hunk 6]
```diff
@@ -120,14 +120,14 @@ workflows:
...
-          context: mattermost-ci-s3
+          s3-copy-to: s3://pr-builds.mattermost.com/mattermost-load-test/
+          context: mattermost-ci-pr-builds-s3
           <<: *only_master
           requires:
             - build-ltops-and-test
       - build-docker:
           <<: *mm_binary
-          loadtest-binary: https://releases.mattermost.com/mattermost-load-test/mattermost-load-test.tar.gz
+          loadtest-binary: https://pr-builds.mattermost.com/mattermost-load-test/mattermost-load-test.tar.gz
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          loadtest-binary: https://releases.mattermost.com/mattermost-load-test/mattermost-load-test.tar.gz
```

File: .circleci/config.yml [hunk 7]
```diff
@@ -120,14 +120,14 @@ workflows:
 
       - upload-s3:
           name: upload-s3-master
-          s3-copy-to: s3://releases.mattermost.com/mattermost-load-test/
-          context: mattermost-ci-s3
+          s3-copy-to: s3://pr-builds.mattermost.com/mattermost-load-test/
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          s3-copy-to: s3://releases.mattermost.com/mattermost-load-test/
```

File: .circleci/config.yml [hunk 8]
```diff
@@ -120,14 +120,14 @@ workflows:
...
+          context: mattermost-ci-pr-builds-s3
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          context: mattermost-release-s3
```

**Review by DSchalla — CHANGES_REQUESTED** [review]
Added some changes, we currently do the following:
- PR goes into the `pr-builds` S3 bucket
- Releases go into the `releases` S3 bucket

The requested changes should reflect that accordingly, but please double check.

File: .circleci/config.yml [hunk 9]
```diff
@@ -104,14 +104,14 @@ workflows:
 
       - upload-s3:
           name: upload-s3-branches
-          s3-copy-to: s3://releases.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/
+          s3-copy-to: s3://pr-builds.mattermost.com/loadtest-pr/${CIRCLE_BRANCH}/
           context: mattermost-ci-s3
```
Comment (DSchalla) [inline .circleci/config.yml]:
```suggestion
          context: mattermost-ci-pr-builds-s3
```

I cannot see that context currently existing, @cpanato was that a legacy one?

Comment (DSchalla) [reply .circleci/config.yml]: (For reference, the `mattermost-ci-s3` doesn't exist, the `mattermost-ci-pr-builds-s3` does)

**Review by cpanato — APPROVED** [review]

**Review by DSchalla — APPROVED** [review]

**Pull request merged**